src/compiler/types.ts

```diff
@@ -1582,14 +1582,15 @@ module ts {
         Tuple                   = 0x00002000,  // Tuple
         Union                   = 0x00004000,  // Union
         Anonymous               = 0x00008000,  // Anonymous
+        Instantiated            = 0x00010000,  // Instantiated anonymous type

```

Comment:
Agreed.  'Instantiated' conveys something entirely different to me.  Furthermore, if it is supposed to mean InstantiatedAnonymousTYpe, then i'm surprised that it doesn't include 0x00008000 as part of it.